src/main/java/reposense/commits/CommitResultAggregator.java

```diff
@@ -129,19 +129,15 @@ private static Date getStartOfDate(Date current) {
     }
 
     private static Date getStartDate(List<CommitResult> commitInfos) {
-        Calendar cal = Calendar.getInstance();
-        cal.set(Calendar.YEAR, 2050);
-        Date min = cal.getTime();
+        Date min = new Date(1970);
```

Comment:
The since date will be defaulted to 1970 if it is set to a future date right?

Not sure if this value is useful. Try comparing it with today in the report?

Reply:
The "since date" will be defaulted to 1970 if user sets "until date" to any date before the first commit